Tidy scivision README

edwardchalstrey1 opened the pull request:
# Changes

Simplifies the repo landing page and moves the maintainer documentation to a separate place

ots22 commented on docs/maintainer_docs.md:
```diff
@@ -0,0 +1,58 @@
+# Maintainers
+
+This document is intended for maintainers of the scivision project and includes the following how to's:
```
how to's -> how-tos

ots22 commented on README.md:
```diff
@@ -78,64 +77,16 @@ Table of contents
-## Maintainers
-
-If you are new to the `scivision` project and wish to become a maintainer for either the PyPi release or the  readthedocs documentation, send an email to scivision@turing.ac.uk
+See our maintainer documentation [here](docs/maintainer_docs.md), which explain how to update the scivision package and documentation.
```
update the scivision PyPI package

ots22 replied:
explain -> explains

ots22 commented on README.md:
```diff
@@ -78,64 +77,16 @@ Table of contents
+## 📚 Documentation
 
-Documentation is available at https://scivision.readthedocs.io/en/latest/
+User documentation for scivision is available at our [readthedocs](https://scivision.readthedocs.io/en/latest/) website.
```
Just 'Documentation' seems more natural to me (rather that 'User documentation')

ots22 commented on docs/maintainer_docs.md:
```diff
@@ -0,0 +1,58 @@
+# Maintainers
```
perhaps call this file 'maintainers.md', 'maintainer_instructions.md' or similar (to avoid the docs suffix - it's in docs/ already!)

ots22 reviewed (changes requested):
This looks good - not cluttering the README with instructions for maintainers is a good idea. 

These files are sphinx doc pages, so don't forget to update index.rst so the changes get reflected there too.

Perhaps we could update some of these links (e.g. contributing, maintainer_docs) so that they point to the relevant place on readthedocs, rather than the markdown file on GH? The idea being to keep the docs conceptually in one place, with the README file a landing page/jumping off point only.

edwardchalstrey1 commented:
> This looks good - not cluttering the README with instructions for maintainers is a good idea.
> 
> These files are sphinx doc pages, so don't forget to update index.rst so the changes get reflected there too.
> 
> Perhaps we could update some of these links (e.g. contributing, maintainer_docs) so that they point to the relevant place on readthedocs, rather than the markdown file on GH? The idea being to keep the docs conceptually in one place, with the README file a landing page/jumping off point only.

Thanks for the review @ots22 I've made most of the changes including linking to the readthedocs contribution page -  actually the maintainer docs I think should be left off readthedocs - what do you think? The index.rst is currently different from README.md in a way I think makes sense. 

I need to update the rst files on the other PR though!

edwardchalstrey1 commented on docs/index.rst:
```diff
@@ -28,7 +28,7 @@ User's Guide
    how_to_install
    contributing
-   release_process
+   maintainers
```
I might have changed my mind, perhaps we should include this in readthedocs

edwardchalstrey1 commented:
> To me this is all part of the project documentation - is there a reason to separate it?

@ots22 perhaps you're right, it just seems odd to have a bit about how to update readthedocs inside readthedocs haha

edwardchalstrey1 commented:
@ots22 would you be happy for me to merge this? I can be persuaded to adopt your view on the maintainer docs if you think it makes the most sense, but see above comment

edwardchalstrey1 commented:
> Ah sorry - I still think it makes sense to keep it all together - even the readthedocs part (perhaps even especially so!)

@ots22 I've moved this to readthedocs and added some links to the github repo (and pypi repo) so that it makes sense in this context now 😄
